widgets/image/tpl/base.php

```diff
@@ -13,6 +13,9 @@
 
 	if(!empty($src[1])) $attr['width'] = $src[1];
 	if(!empty($src[2])) $attr['height'] = $src[2];
+	if (function_exists('wp_get_attachment_image_srcset')) {
+		$attr['srcset'] = esc_attr(wp_get_attachment_image_srcset($instance['image'], $instance['size']));
```

Comment:
`esc_attr` isn't necessary here as it's called on all the values of the `$attr` array just before being output below.